### PR TITLE
Adding HostMeetingTest

### PR DESCRIPTION
**Pull request by gavatesnehal**
one Test case is added

### CODE REVIEW

File: src/test/java/com/Testcase/HostMeetingPageTest.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+public class HostMeetingPageTest {
+
+	UIKeywords keyword = UIKeywords.getInstance();
+
+	@Test
+	public void validHostMeetingTestCases() {
+		HostMeetingPage page = PageFactory.initElements(keyword.driver, HostMeetingPage.class);
+		page.clickOnHostMeetingTab();
+	}
```
Comment (harshalilokhande) [inline src/test/java/com/Testcase/HostMeetingPageTest.java:17]: add more methods

File: src/main/java/com/testingshastra/pages/HostMeetingPage.java [hunk 2]
```diff
@@ -10,9 +10,11 @@
 	UIKeywords key = UIKeywords.getInstance();
 	@FindBy(css = "div#navbarContent li:nth-child(2)")
 	public WebElement hostMeetingTab;
+	
+	
 
-//@FindBy(css="div#navbarContent li:nth-child(2)")
-//public WebElement ;
+@FindBy(css="a.button--light")
```
Comment (harshalilokhande) [inline src/main/java/com/testingshastra/pages/HostMeetingPage.java:16]: use another locator

**Review by harshalilokhande — CHANGES_REQUESTED** [review]

File: src/test/java/com/Testcase/HostMeetingPageTest.java (hunk 1 above)
Comment (gavatesnehal) [reply src/test/java/com/Testcase/HostMeetingPageTest.java:17]: no need to add extra methods

Comment (harshalilokhande) [reply src/test/java/com/Testcase/HostMeetingPageTest.java:17]: ok

**Review by harshalilokhande — APPROVED** [review]

**Pull request merged**